scripts/check-autodoc.py

```diff
@@ -0,0 +1,77 @@
+"""
+Author: Eric J. Ma, borrowing code from Samuel Oranyeli.
+This Python script checks the docs pages for functions
+that have not been added to the "General Functions" API docs.
+
+Only the "general_functions.rst" needs the checks,
+as the others are auto-populated.
+This situation simply evolved out of how the docs are structured,
+I am not sure exactly how it turned out this way. :)
+"""
+
+import re
+from pathlib import Path
+from typing import List, Tuple
+
+
+def extract_function_names(
+    test_folder: Path, exclude_names: List[str]
+) -> List[str]:  # skipqc
+    """
+    Extract function names from the list of functions.
```

Comment:
Do you want to put this docstring on one line following PEP 257?

Reply:
I think I just let black do whatever it needed to.

Reply:
Got it. I believe that `black` doesn't touch docstrings and doesn't have an opinion on them. (I tested with a 300 character long docstring on a single line)

In the grand scheme of things, I think this is a minor aspect of docstrings, but might be worth trying to align the codebase.

Following PEP 257, we could put one-liners on one line, and 

>Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.

I think I personally notice the different styles in the codebase when scanning through functions, and could just do an issue/PR myself to align them to whichever style is decided.

Reply:
Good point. Consistency does matter. Pushing up a change.